tests/pytests/unit/modules/test_win_useradd.py

```diff
@@ -0,0 +1,344 @@
+import pytest
+from saltfactories.utils import random_string
+
+import salt.modules.cmdmod
+import salt.modules.win_useradd as user
+import salt.utils.data
+from salt.exceptions import CommandExecutionError
+
+pytestmark = [
+    pytest.mark.destructive_test,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.windows_whitelisted,
+]
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {user: {"__salt__": {"cmd.run_all": salt.modules.cmdmod.run_all}}}
+
+
+@pytest.fixture
+def username_str():
+    _username = random_string("test-account-", uppercase=False)
+    try:
+        yield _username
+    finally:
+        try:
+            user.delete(_username, purge=True, force=True)
+        except Exception:  # pylint: disable=broad-except
+            # The point here is just system cleanup. It can fail if no account was created
```

Comment:
If we actually create accounts, then move this to functional please.

Reply:
Functional was a lot slower

Reply:
Moved to functional, but kept the `configure_loader_modules` fixture